### PR TITLE
Fixing replication

### PR DESCRIPTION
**Pull request by manasnisar**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/dbs/event/authorizer_aggregate.go [hunk 1]
```diff
@@ -56,18 +54,10 @@ func (a *AuthorizerAggregate) SetTotalRewards(value currency.Coin) {
 	a.TotalRewards = value
 }
 
-func (edb *EventDb) ReplicateAuthorizerAggregate(p common.Pagination) ([]AuthorizerAggregate, error) {
+func (edb *EventDb) ReplicateAuthorizerAggregate(round int64, limit int, offset int) ([]AuthorizerAggregate, error) {
 	var snapshots []AuthorizerAggregate
-
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/dbs/event/authorizer_aggregate.go:61]: let's refactor and remove this duplication for different providers

Comment (dabasov) [reply code/go/0chain.net/smartcontract/dbs/event/authorizer_aggregate.go:61]: extract common method where we can pass table name and structure to scan into

**Review by dabasov — CHANGES_REQUESTED** [review]

**Review by dabasov — APPROVED** [review]
lgtm

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**